src/site/topics/recursion/recursion.rst

```diff
@@ -98,59 +120,72 @@ Lists
     ``a + b + c + d + e + []``
 
 
-Recursion
-=========
+
+Repeating Patterns
+==================
 
 * In counting example, it may feel like cheating by saying "1 + however many are after the front"
-    * *however many are after the front* seems like we're skipping a step
 
-* However, with the list example, we also see the we repeatedly apply the same rule over and over again on smaller and smaller lists until we get to the end (the empty list)
+    * "however many are after the front" seems like skipping a step
+
+
+* But, with the list example, it was observed that repeatedly applying the same rule over and over on smaller and smaller lists resulted in hitting an *end*
+
+    * The empty list
+    * The base case
     * This pattern arises a lot with recursion --- repeatedly apply the same rules on slightly different versions of the problem
 
-* As mentioned earlier, we're typically going to have a **base case** and a **recursive case**
-* In fact, we can have multiple base cases and recursive cases
-    * We will see examples of these when discussing trees
+
+* As mentioned earlier, there is typically going to be a *base case* and a *recursive case*
+* In fact, there can be multiple base cases and recursive cases
+
+    * Several examples of this will be seen when discussing trees
 
 
 Going Up and Down
 -----------------
 
 * The set of natural numbers :math:`\mathbb{N}` can be recursively defined as:
+
     * 0 is a natural number
-    * A natural number + 1 is a natural number
+    * A natural number :math:`+ 1` is a natural number
```

Comment:
I get annoyed by this example, because we use (+1) despite not knowing 1 is a natural.

It's fine. I have to deal with my own personal demons.